### DISCUSSION TITLE
Wanting to start using rook-ceph on my stock Ubuntu partition

### DISCUSSION THREAD

**Discussion by budimanjojo** [rook/rook#8908] [Q&A] [answered]
Hello, I have a default partitioning scheme from ubuntu 20.04 server install. Here is my current `lsblk` output:
```
budiman@budimanjojo-kmaster3:~$ lsblk
NAME                      MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
loop0                       7:0    0  55.4M  1 loop /snap/core18/2128
loop1                       7:1    0  67.3M  1 loop /snap/lxd/21545
loop2                       7:2    0  32.3M  1 loop /snap/snapd/12704
loop3                       7:3    0  61.8M  1 loop /snap/core20/1081
loop4                       7:4    0  32.3M  1 loop /snap/snapd/13170
loop5                       7:5    0  70.3M  1 loop /snap/lxd/21029
sda                         8:0    0 111.8G  0 disk
├─sda1                      8:1    0   512M  0 part /boot/efi
├─sda2                      8:2    0     1G  0 part /boot
└─sda3                      8:3    0 110.3G  0 part
  └─ubuntu--vg-ubuntu--lv 253:0    0  55.1G  0 lvm  /
```
As you can see, `/dev/sda3` is a PV with half the size of it being used for root partition.
My question is:
- Can I just create a new LV inside /dev/sda3 and deploy `rook-ceph` without doing anything else?
Thanks before :)

**Comment by satoru-takeuchi** [comment, marked as answer]
> Can I just create a new LV inside /dev/sda3 and deploy rook-ceph without doing anything else?
Thanks before :)

You can add this lv in your cluster if you use pvc-based cluster. And then a PVC corresponding to this lv should be created. After that, when count field in StorageCalssDeviceSets field is set to the desired number of osds, this lv will be added to your cluster.

**Reply by budimanjojo** [reply]
Thanks for the reply. If I want to use normal cluster, what should I do? Resize the PV and create a new PV and unformatted LV?

**Reply by satoru-takeuchi** [reply]
If normal cluster means host-based cluster, which you specify devices fir OSDs in CephCluster CR, raw device or partition is needed.

**Reply by budimanjojo** [reply]
Got it. I don't have extra disk, so I'll go with partition. Thank you :)
